### PR TITLE
[CARBONDATA-2295] Added property for UnsafeMemory configuration

### PR DESCRIPTION
**Pull request by bhavya411**
Added property for setting up the unsafe memory limit in MB

 - [ X] Testing done
       Added a new property in the test cases to make sure they all run fine
        Tested manually on the presto client as well

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/3442/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/4668/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4180/

File: integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java [hunk 1]
```diff
@@ -366,7 +368,10 @@ public TBase create() {
   public List<CarbonLocalInputSplit> getInputSplits2(CarbonTableCacheModel tableCacheModel,
                                                      Expression filters)  {
     List<CarbonLocalInputSplit> result = new ArrayList<>();
-
+    if(config.getUnsafeMemoryInMb() != null) {
+      CarbonProperties.getInstance().addProperty(CarbonCommonConstants.UNSAFE_WORKING_MEMORY_IN_MB,
```
Comment (chenliang613) [inline integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java]: please add this configuration description to presto/README.md

Comment (bhavya411) [reply integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonTableReader.java]: Done

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/3453/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/4680/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/4189/

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**